packages/errors/package.json

```diff
@@ -21,9 +21,10 @@
     "strip-ansi": "6.0.0"
   },
   "devDependencies": {
+    "@packages/config": "0.0.0-development",
     "@packages/ts": "0.0.0-development",
     "@types/chai": "4.2.15",
-    "@types/mocha": "8.2.2",
+    "@types/mocha": "^8.0.3",
```

Comment:
why did we downgrade mocha?

Reply:
Only the types and this is because types have to be consistent to avoid conflict with cypress.